src/ImpactEvaluator.sol

```diff
@@ -159,6 +159,10 @@ contract ImpactEvaluator is AccessControl {
 
         for (uint i = 0; i < addresses.length; i++) {
             address payable addr = addresses[i];
+            if (addr == 0x000000000000000000000000000000000000dEaD) {
```

Comment:
Should we filter out our placeholder address `0xf100Ac342b7DE48e5c89f7029624eE6c3Cde68aC` too?

https://github.com/filecoin-station/spark-evaluate/blob/87914366bc0adbde87d1d6f444c7d620b1a3e471/lib/preprocess.js#L82-L86

Reply:
Good catch, what about we instead change that address to the burner address too?

Reply:
> we instead change that address to the burner address too?

SGTM 👍🏻 

Could you make that change yourself, please?

Reply:
https://github.com/filecoin-station/spark-evaluate/pull/40